h2o-core/src/test/java/water/parser/ParseTimeTest.java

```diff
@@ -9,7 +9,10 @@
 import water.fvec.*;
 
 public class ParseTimeTest extends TestUtil {
-  @BeforeClass static public void setup() { stall_till_cloudsize(1); }
+  @BeforeClass static public void setup() {
+    stall_till_cloudsize(1);
+    ParseTime.setTimezone("America/Los_Angeles"); // by default, we now parse in UTC
```

Comment:
It is better to change the expected results than to force the timezone. Forcing the timezone is a side-effect and can affect other tests.

Reply:
The tests all run independently, no?

Reply:
They don't. They run on the same cluster that is setup once.